### DISCUSSION TITLE
Not sure how to configure outgoing

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#14262] [How to] [answered]
When I calling email template in workflow, it always showing that email coming from "YetiForce Mailer"
In SMTP Configuration I am unable to set Sender Name as YF returning: 

Error!!!
LBL_NO_EMAIL_TEMPLATE

Bug or missconfiguration?

**Comment by apcloic** [comment, marked as answer]
@waran70 
My guess is that you have deleted "Test mail about the mail server configuration." email template.
https://gitstable.yetiforce.com/index.php?module=EmailTemplates&view=Detail&record=95

Regarding XMailer,Hostname and FromName, it's hardcoded in 
https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/app/Mailer.php
````
		$this->mailer->XMailer = 'YetiForceCRM Mailer';

		$this->mailer->Hostname = 'YetiForceCRM';

		$this->mailer->FromName = 'YetiForce Mailer';


````

I don't know if YF licence allows us to change this.
Regards,

**Reply by waran70** [reply]
I did replayed to myself with solution but it not showing here somehow. Yes that was templates issue.

**Comment by davide-alghi** [comment]
Hi @waran70,
it looks like different kind of problems

>
> When I calling email template in workflow, it always showing that email coming from "YetiForce Mailer"
>

if recipient receives email with sender name "YetiForce Mailer", that means you didn't set "Sender name" field in SMTP configuration form

>
> In SMTP Configuration I am unable to set Sender Name as YF
>

what does it means? do you insert "Sender name", but isn't it saved?

>
> Error!!!
> LBL_NO_EMAIL_TEMPLATE
>

when and where do you get this message? this message notify that the email template doesn't exist: it has no relation with SMTP
pls check if the email template, you set in the workflow task, exists and it has Type = Record


my additional notes:

- LBL_NO_EMAIL_TEMPLATE label in not translated (@mariuszkrzaczkowski)
- if SMTP is correctly configured everything works fine (YF 5.3 and YF 6.1 - both with last SPs)
- there it can be set SMTP in email template and in workflow task: master is SMTP in workflow task
